server/src/core/symbols/symbol.rs

```diff
@@ -3188,3 +3177,35 @@ impl Symbol {
         res
     }*/
 }
+
+impl SymbolPath {
+    pub fn as_single(self) -> String {
+        match self {
+            Self::Single(s) => s,
+            _ => panic!("Not a Single"),
+        }
+    }
+
+    pub fn is_single(&self) -> bool {
+        matches!(self, Self::Single(_))
+    }
+
+    pub fn as_multiple(self) -> Vec<String> {
+        match self {
+            Self::Multiple(v) => v,
+            _ => panic!("Not a Multiple"),
+        }
+    }
+
+    pub fn is_multiple(&self) -> bool {
+        matches!(self, Self::Multiple(_))
+    }
+
+    pub fn as_vec(self) -> Vec<String> {
```

Comment:
This enables us to use the result of `.path()` just like the old `paths()` - which is probably a bad thing. :grimacing: